ui/src/app/components/card.tsx

```diff
@@ -9,10 +9,22 @@ export const styles = reactStyles({
     border: '1px solid #d7d7d7',
     display: 'flex',
     flexDirection: 'column',
-    width: '12rem',
```

Comment:
these were always overwritten it turns out and we never had any cards with these sizing dimensions -____-